File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1173.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

v = parseInt(lines.shift())
ant = v
for(i=0;i<10;i++){
    if(i!=0){
        ant = ant*2
    }
    console.log("N["+i+"] = "+ant)
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1173.c
#include <stdio.h>

int main(){
    int i, v, ant;
    scanf("%d", &v);
    for(i=0;i<10;i++){
        if(i==0){
            ant = v;
        } else{
            ant = ant*2;
        }
        printf("N[%d] = %d\n", i, ant);
    }

    return 0;
}